src/main/java/seedu/address/ui/ContactDetails.java

```diff
@@ -91,12 +90,14 @@ private void setPanelInformation() {
                 Label notesHeader = new Label("Notes");
                 notesHeader.setId("notes-header");
                 notesList.getChildren().add(notesHeader);
+
+                int[] index = {1};
                 person.getNotes().stream()
-                        .sorted(Comparator.comparing(note -> note.getNote()))
                         .forEach(note -> {
-                            Label label = new Label("• " + note.getNote());
+                            Label label = new Label(index[0] + ". " + note.getNote());
```

Comment:
`AtomicInteger` can use index.getAndIncrement(), so no longer need to increment at the end.

Reply:
alright, I will update it